lib/assets/components/explore_tiles.dart

```diff
@@ -320,67 +321,119 @@ class ExploreNewsArticleTile extends ExploreTile {
+                                  UserProgressState>(
+                                builder: (context, state) {
+                                  final isComplete =
+                                      state.newsArticleIsCompleted(article.id);
+                                  if (isComplete) {
+                                    return _ExploreTileCheckmark(
+                                      completed: isComplete,
+                                    );
+                                  }
+                                  return Container();
+                                },
+                              ),
+                            ),
+                          ],
+                        ),
+                      ),
                     ),
-                    Text(
-                      article.subtitle,
-                      maxLines: 2,
-                      overflow: TextOverflow.ellipsis,
-                      style: Theme.of(context)
-                          .textTheme
-                          .bodySmall!
-                          .apply(fontStyle: FontStyle.normal),
+                    Expanded(
+                      child: Padding(
+                        padding: const EdgeInsets.all(8),
+                        child: Column(
+                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            Text(
+                              article.title,
+                              style:
+                                  Theme.of(context).textTheme.headlineMedium!,
+                              textScaler: const TextScaler.linear(.7),
+                              maxLines: 2,
+                              overflow: TextOverflow.ellipsis,
+                            ),
+                            Text(
+                              article.subtitle,
+                              maxLines: 2,
+                              overflow: TextOverflow.ellipsis,
+                              style: Theme.of(context)
+                                  .textTheme
+                                  .bodySmall!
+                                  .apply(fontStyle: FontStyle.normal),
+                            ),
+                            Text(
+                              article.publishedAt
+                                  .let(DateFormat('d MMM y').format),
+                            ),
+                          ],
+                        ),
+                      ),
                     ),
-                    Text(
-                      article.publishedAt.let(DateFormat('d MMM y').format),
+                    Ink(
+                      padding: const EdgeInsets.symmetric(vertical: 8),
+                      decoration: const BoxDecoration(
+                        color: Color.fromRGBO(247, 248, 252, 1),
```

Comment:
Can these colors be extracted to theme? Or if this is specific to file maybe extracted to `const` variable